avm/res/api-management/service/README.md

```diff
@@ -1,10 +1,5 @@
 # API Management Services `[Microsoft.ApiManagement/service]`
 
-> ⚠️THIS MODULE IS CURRENTLY ORPHANED.⚠️
```

Comment:
Waiting for @tony-box membership to relevant avm groups

Reply:
@eriqua The membership is updated

Reply:
Super, I'll ask @tony-box to kindly approve :)